src/components/Navbar/__test__/__snapshots__/Navbar.test.js.snap

```diff
@@ -280,29 +287,42 @@ exports[`renders correctly 1`] = `
         className="flex gap-3 capitalize flex-col md:flex-row"
       >
         <a
-          href="/components/Signup"
+          href="/signin"
           onClick={[Function]}
           onMouseEnter={[Function]}
           onTouchStart={[Function]}
         >
           <div
             className="bg-purple-light py-2 px-5 text-white rounded-3xl text-sm hover:bg-violet-800 transition-all cursor-pointer"
           >
-            sign up
+            sign in
           </div>
         </a>
-        <div
-          className="bg-purple-light py-2 px-5 text-white rounded-3xl text-sm hover:bg-violet-800 transition-all cursor-pointer"
+        <a
+          href="/sell-items"
+          onClick={[Function]}
+          onMouseEnter={[Function]}
+          onTouchStart={[Function]}
         >
-          sell items
-        </div>
+          <div
+            className="bg-purple-light py-2 px-5 text-white rounded-3xl text-sm hover:bg-violet-800 transition-all cursor-pointer"
+          >
+            sell items
+          </div>
+        </a>
       </div>
       <div
         className="text-2xl text-orange-600 hover:text-orange-700 transition"
       >
-        <button>
+        <a
+          href="/cart"
+          onClick={[Function]}
+          onMouseEnter={[Function]}
+          onTouchStart={[Function]}
+        >
            
           <svg
```

Comment:
This should not be used like this, instead extract the code into a separate svg file and then embed it.

Reply:
snapshot too 😂